src/utils/tests/numbers.test.ts

```diff
@@ -0,0 +1,3594 @@
+/* eslint-disable @typescript-eslint/no-loss-of-precision */
+import { Currency, CurrencyAmount, Fraction, Percent, Price, Token } from '@kyberswap/ks-sdk-core'
+import { afterAll, beforeAll, describe, expect, test, vi } from 'vitest'
+
+import { formatDisplayNumber } from '../numbers'
+
+describe('formatDisplayNumber tests', () => {
+  beforeAll(() => {
+    vi.mock('uuid', async importOriginal => {
+      const mod = (await importOriginal()) as any
+      return {
+        ...mod,
+        v4: () => '',
+      }
+    })
+  })
+  afterAll(() => {
+    vi.clearAllMocks()
+  })
+  describe('decimal', () => {
+    describe('number', () => {
+      describe('large numbers', () => {
+        describe('2 significantDigits', () => {
+          test('format number 1 correctly', () => {
```

Comment:
testcase description should be like this: 
format number 1 which has significant number < 2

Reply:
It would be quite hard to naming all testcases like this.